support/kind/kind.go

```diff
@@ -110,7 +110,7 @@ func (k *Cluster) WithOpts(opts ...support.ClusterOpts) support.E2EClusterProvid
 func (k *Cluster) getKubeconfig() (string, error) {
 	kubecfg := fmt.Sprintf("%s-kubecfg", k.name)
 
-	p := utils.RunCommand(fmt.Sprintf(`%s get kubeconfig --name %s`, k.path, k.name))
+	p := utils.RunCommand(fmt.Sprintf(`%s get kubeconfig --name %s -q`, k.path, k.name))
```

Comment:
@heylongdacoder my only concern here is that if there are other problems with retrieving the kubeconfig, that command will also stay quiet. This can be bad downstream for a running test, causing it to fail in mysterious ways since that command appears to be working when in fact, there are errors.